From 248116fcceff5e76468711647ab5102dd143a00a Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Wed, 6 Sep 2017 10:55:58 +0300
Subject: [PATCH 1235/1345] doc: mvebu: a80x0: fix typo on DPR for
 configuration D

commit  e1e45d93d5d3ebfb9a928ca34db4cd40d6b0de13 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Change-Id: Idb2e72db768934dd21b002f60559087aa445d3ed
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43808
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
(cherry picked from commit 072290631df529b358128a23417f80f34aaef414)
Reviewed-on: http://vgitil04.il.marvell.com:8080/44027
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../mvebu/a7k-a8k/armada8040-db-setup.txt          |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt b/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
index 4e5bb6a..6e6b0f4 100644
--- a/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
+++ b/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
@@ -60,7 +60,7 @@ DPR Setting
 | Jumper-module (CON44): SLM1564		| DPR 96-108 = 2-3		| DPR 109-112 = 2-3, others = 1-2	|
 | Serdes CP0 site module (CON42): SLM1521	| DPR 2-5 = 1-2			| DPR 2-5 = 1-2				|
 | Serdes CP1 site module (CON41): SLM1521	| DPR 2-5 = 1-2			| DPR 2-5 = 1-2				|
-| DB8040					| DPR 76,87,87 = 2-3		| DPR 81 = 2-3, DPR 119,121,123-126 =1-2|
+| DB8040					| DPR 76,87,88 = 2-3		| DPR 81 = 2-3, DPR 119,121,123-126 =1-2|
 |						| Audio module (CON51): SLM1449	|					|
 
 The tables below summarizes the interface configuration of each setup
-- 
1.7.9.5

